test/test-framework/deploy/olm-catalog/memcached-operator/0.0.2/memcached-operator.v0.0.2.clusterserviceversion.yaml

```
@@ -1,30 +1,109 @@
-# This file defines the ClusterServiceVersion (CSV) to tell the catalog how to display, create and
-# manage the application as a whole. If changes are made to the CRD for this application kind,
-# make sure to replace those references below as well.
 apiVersion: operators.coreos.com/v1alpha1
 kind: ClusterServiceVersion
 metadata:
   annotations:
-    alm-examples: '[{"apiVersion":"cache.example.com/v1alpha1","kind":"Memcached","metadata":{"name":"example-memcached"},"spec":{"size":3}}]'
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "cache.example.com/v1alpha1",
+          "kind": "Memcached",
+          "metadata": {
+            "name": "example-memcached"
+          },
+          "spec": {
+            "size": 3
+          }
+        },
+        {
+          "apiVersion": "cache.example.com/v1alpha1",
+          "kind": "MemcachedRS",
+          "metadata": {
+            "name": "example-memcachedrs"
+          },
+          "spec": {
+            "numNodes": 4
+          }
+        }
+      ]
     capabilities: Basic Install
   name: memcached-operator.v0.0.2
   namespace: placeholder
 spec:
-  installModes:
-  - type: OwnNamespace
-    supported: true
-  - type: SingleNamespace
-    supported: true
-  - type: MultiNamespace
-    supported: false
-  - type: AllNamespaces
-    supported: true
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - kind: MemcachedRS
+      name: memcachedrs.cache.example.com
+      version: v1alpha1
+    - description: Represents a cluster of Memcached apps
+      displayName: Memcached App
+      kind: Memcached
+      name: memcacheds.cache.example.com
+      resources:
+      - kind: Deployment
+        name: ""
+        version: v1
+      - kind: ReplicaSet
+        name: ""
+        version: v1
+      - kind: Pod
+        name: ""
+        version: v1
+      specDescriptors:
+      - description: The desired number of member Pods for the deployment.
+        displayName: Size
+        path: size
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podCount
+      statusDescriptors:
+      - description: The current status of the application.
+        displayName: Status
+        path: phase
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase
+      - description: Explanation for the current status of the application.
+        displayName: Status Details
+        path: reason
+        x-descriptors:
+        - urn:alm:descriptor:io.kubernetes.phase:reason
+      version: v1alpha1
```

```suggestion
      version: v1alpha1
    - kind: NotExistKind
      name: notexist.example.com
      version: v1alpha1

```
should fix the scorecard error